src/Components/Aspire.Microsoft.EntityFrameworkCore.SqlServer/AspireSqlServerEFCoreSqlClientExtensions.cs

```diff
@@ -96,32 +79,6 @@ public static class AspireSqlServerEFCoreSqlClientExtensions
                 static hcBuilder => hcBuilder.AddDbContextCheck<TContext>());
         }
 
-        void ConfigureDbContext(DbContextOptionsBuilder dbContextOptionsBuilder)
-        {
-            // We don't register logger factory, because there is no need to:
-            // https://learn.microsoft.com/dotnet/api/microsoft.entityframeworkcore.dbcontextoptionsbuilder.useloggerfactory?view=efcore-7.0#remarks
-            dbContextOptionsBuilder.UseSqlServer(settings.ConnectionString, builder =>
-            {
-                if (string.IsNullOrEmpty(settings.ConnectionString))
-                {
-                    throw new InvalidOperationException($"ConnectionString is missing. It should be provided in 'ConnectionStrings:{connectionName}' or under the 'ConnectionString' key in '{DefaultConfigSectionName}' or '{typeSpecificSectionName}' configuration section.");
-                }
-
-                // Resiliency:
-                // Connection resiliency automatically retries failed database commands
-                if (settings.MaxRetryCount > 0)
-                {
-                    builder.EnableRetryOnFailure(settings.MaxRetryCount);
```

Comment:
I thought that Aspire would still enable retries either through `EnableRetryOnFailure()` or `UseAzureSqlDefaults()`